src/mscorlib/corefx/System/Globalization/CultureData.Windows.cs

```diff
@@ -567,5 +567,25 @@ private static string LCIDToLocaleName(int culture)
         {
             throw new NotImplementedException();
         }
+        
+        private static int GetAnsiCodePage(string cultureName)
+        {
+            throw new NotImplementedException();
```

Comment:
@weshaggard this is temporary til we get the right implementation for .Net Native.